mkosi/kmod.py

```diff
@@ -17,10 +18,25 @@
 
 def loaded_modules() -> list[str]:
     # Loaded modules are listed with underscores but the filenames might use dashes instead.
-    return [
-        rf"/{line.split()[0].replace('_', '[_-]')}\.ko"
-        for line in Path("/proc/modules").read_text().splitlines()
-    ]
+    return [normalize_module_name(line.split()[0])
+            for line in Path("/proc/modules").read_text().splitlines()]
+
+
+def globs_match_module(name, globs):
```

Comment:
Just for completeness, did you consider simply postprocessing the globs to regex via `fnmatch.translate`? That would keep the rest more or less the same?

Reply:
It's also what Python would do behind the scene anyhow

Reply:
I don't see any particular reason to do this. This code runs fast enough to not be noticeable. And in the longer term I hope to get rid of all the regexp-based patterns.